web/themes/custom/hatter/imgs/README.md

```diff
@@ -0,0 +1,3 @@
+<h1>Images</h1>
```

Comment:
Seems like this and a whole bunch of the theme things maybe shouldn't have been committed?

Reply:
I'm not sure. I think it was being committed and then it was removed and then I think it got re-added. @zendoodles this should be removed correct?